### DISCUSSION TITLE
vulnerability data

### DISCUSSION THREAD

**Discussion by andrewdwalker** [emgucv/emgutf#93] [General]
We are suddenly getting this error:
4>C:\Andrew\Reporter.csproj : warning NU1900: Error occurred while getting package vulnerability data: Unable to load the service index for source ...
I'll also mention that going to ... from a browser gives "Bad Gateway"
Questions:
1.  What is vulnerability data?  Is it new for emgu?  
2.  Is there a separate web address for the actual Nuget package?  That is, am I only having problems with vulnerability data?
3.  Can I "turn off"/disable vulnerability data?

**Comment by emgucv** [comment]
Thanks for pointing that out. It was a server issue and has been fixed now. Please try again. I should be working now.

Btw, I have removed the link to the commercial nuget repository in your original post. Thanks again for reporting.

**Comment by andrewdwalker** [comment]
Looks like your server might be down again.
Regards,
Andrew


    On Friday, September 27, 2024 at 08:05:20 PM PDT, Emgu ***@***.***> wrote:  
 
 


Thanks for pointing that out. It was a server issue and has been fixed now. Please try again. I should be working now.

Btw, I have removed the link to the commercial nuget repository in your original post. Thanks again for reporting.

**Reply by emgucv** [reply]
We were performing a server update. 

It is done and the website should be back to normal now.
